### ISSUE TITLE
fix minimal installation proftpd postfix

### ISSUE THREAD

**Issue by aryanrajseo** [virtualmin/virtualmin-lamp-stack-ubu#4]
as discussed here 

https://forum.virtualmin.com/t/virtualmin-installation-error-on-almalinux-and-rockylinux/131195

https://forum.virtualmin.com/t/virtualmin-installation-error-on-almalinux-and-rockylinux/131195/8?u=aryan

**Comment by iliaross** [comment]
Hello,

Thanks for reporting it!

This issue will be fixed after [Joe](@swelljoe) deploys all the recent changes.

**Comment by aryanrajseo** [comment]
my apologies for trigging this again, will it get updated here https://github.com/virtualmin/virtualmin-install ?

my servers are still idle waiting for the fix. Thanks. :)

**Comment by iliaross** [comment]
I will tag the new releases now, and if Joe has time today, he will roll them all out.

**Comment by aryanrajseo** [comment]
it is released but not updated on the server. it is still old virtualmin-install.sh

`https://software.virtualmin.com/gpl/scripts/virtualmin-install.sh`

**Comment by iliaross** [comment]
Yes, that’s right! Joe wants to run more thorough testing before releasing it. It will take a while! Sorry!

Merry Christmas and Happy New Year!

**Comment by iliaross** [comment]
Repo suppressed by:

https://github.com/virtualmin/virtualmin-stack-meta
